exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/AbstractParquetScanBatchCreator.java

```diff
@@ -108,7 +108,7 @@ protected ScanBatch getBatch(ExecutorFragmentContext context, AbstractParquetRow
 
         if (!context.getOptions().getBoolean(ExecConstants.PARQUET_NEW_RECORD_READER)
             && !ParquetReaderUtility.containsComplexColumn(footer, rowGroupScan.getColumns())) {
-          logger.debug("Query {} qualifies for new Parquet reader",
+          logger.debug("Query {} qualifies for ParquetRecordReader",
```

Comment:
IMO, it will be much better to log `PARQUET_NEW_RECORD_READER` option and whether or not there are complex columns prior to creating a reader and reader instance class after the reader is created:
```
log.debug("PARQUET_NEW_RECORD_READER is {}. Complex columns {}.", "enabled"/"disabled", "found"/"not found");
Reader reader;
if (...) {
  reader = new ParquetRecordReader()l
} else {
  reader = new DrillParquetReader();
}
log.debug("Query {} uses {});
readers.add(reader);
```

Reply:
Thanks for the suggestion. Done.

Reply:
@vrozov I realised that your proposed wording sounds better. 'not enforced' may cause confusion too. Reverted back to 'enabled/disabled'.